decorators.rb

```diff
@@ -0,0 +1,24 @@
+require './nameable'
+
+class Decorator < Nameable
+  def initialize(nameable)
+    super
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name
+  end
+end
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.chars.map(&:upcase).join
```

Comment:
- [ ] Please ensure that in your line 16, you use `@nameable.correct_name.capitalize` in order to Capitalize only the first letter as part of the project requirement, as right now it capitalizes all the characters.